lib/openstudio-standards/refrigeration/information.rb

```diff
@@ -0,0 +1,140 @@
+module OpenstudioStandards
+  # The Refrigeration module provides methods to create, modify, and get information about refrigeration
+  module Refrigeration
+    # @!group Information
+    # Methods to get information about model refrigeration
+
+    # Find the thermal zone that is best for adding refrigerated display cases into.
+    # First, check for space types that typically have refrigeration.
+    # Fall back to largest zone in the model if no typical space types are found.
+    #
+    # @param model [OpenStudio::Model::Model] OpenStudio model object
+    # @return [OpenStudio::Model::ThermalZone] returns a thermal zone if found, nil if not.
+    def self.refrigeration_case_zone(model)
+      # Ideally, look for one of the space types
+      # that would typically have refrigeration.
+      display_case_zone = nil
+      display_case_zone_area_m2 = 0.0
+      model.getThermalZones.each do |zone|
+        space_type = OpenstudioStandards::ThermalZone.thermal_zone_get_space_type(zone)
+        next if space_type.empty?
+
+        space_type = space_type.get
+        next if space_type.standardsSpaceType.empty?
+        next if space_type.standardsBuildingType.empty?
+
+        stds_spc_type = space_type.standardsSpaceType.get
+        stds_bldg_type = space_type.standardsBuildingType.get
+        case "#{stds_bldg_type} #{stds_spc_type}"
+        when 'PrimarySchool Kitchen',
+            'SecondarySchool Kitchen',
+            'SuperMarket Sales',
+            'QuickServiceRestaurant Kitchen',
+            'FullServiceRestaurant Kitchen',
+            'LargeHotel Kitchen',
+            'Hospital Kitchen',
+            'EPr Kitchen',
+            'ESe Kitchen',
+            'Gro GrocSales',
+            'RFF StockRoom',
+            'RSD StockRoom',
+            'Htl Kitchen',
+            'Hsp Kitchen'
```

Comment:
The typical_walkin/case data has the building_types and space_types where they should go; it seems like that information should be used in this method if available, rather than these hard-coded values.

Reply:
Method now pulls the stds bldg / space type from the typical_refrigeration_x .csvs